### PR TITLE
pfe-cta : fix functions 

### PR DESCRIPTION
**Pull request by starryeyez024**
* Currently the compiled CSS has color functions in it:
https://github.com/patternfly/patternfly-elements/blob/v1.0.0-prerelease.25/elements/pfe-cta/pfe-cta.css#L33

---

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Added interpolation syntax around functions so they compile

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1.  Copile code, browser test, test broadcast vars

#### Browser requirements
Your component should work in all of the following environments:

- [x] Latest 2 versions of Edge
- [x] Internet Explorer 11 (should be useable, not pixel perfect)
- [ ] Latest 2 versions of Firefox (one on Mac OS, one of Windows OS)
- [x] Firefox 60.7.2 (or latest version for Red Hat Enterprise Linux distribution)
- [x] Latest 2 versions of Chrome (one on Mac OS, one of Windows OS)
- [x] Latest 2 versions of Safari
- [ ] Galaxy S9 Firefox
- [ ] iPhone X Safari
- [ ] iPad Pro Safari
- [ ] Pixel 3 Chrome

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Did browser testing pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [ ] Was this feature demo'd and the design review approved?
- [ ] Did you update the CHANGELOG.md file with a summary of this update?

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

File: elements/pfe-cta/src/pfe-cta.scss [hunk 1]
```diff
@@ -9,86 +9,96 @@ $pfe-cta--Color--fallback: #003366;
...
 
+  //  --pfe-cta__arrow--Padding:  0 .125rem 0 .375rem;
+  //calc( #{pfe-local($cssvar: Padding, $region: arrow)}
   .pfe-cta--arrow {
-    display: pfe-local($cssvar: arrow--Display, $region: arrow);
-    padding: pfe-local($cssvar: Padding, $region: arrow);
+    display: #{pfe-local($cssvar: arrow--Display, $region: arrow)};
+    padding: 0;
+    padding-right: calc( 13px + #{var(--pfe-cta__arrow--Padding)} );
+    margin-right:  calc( -13px - #{var(--pfe-cta__arrow--Padding)} );
```
Comment (starryeyez024) [inline elements/pfe-cta/src/pfe-cta.scss]: this part is the WIP, i can back it out. Was just fussing with negative margins to see if we could solve the wrapping arrow this way

File: elements/pfe-cta/src/pfe-cta.scss [hunk 2]
```diff
@@ -9,86 +9,96 @@ $pfe-cta--Color--fallback: #003366;
...
-    color: pfe-local(Color--hover) !important; // because of redhat-theme styles on links
+    color: #{pfe-local(Color--hover)} !important; // because of redhat-theme styles on links
     &::before  {
       border: 1px solid pfe-local($cssvar: BorderColor--hover, $region: inner);
     }
   }
 
   .pfe-cta--arrow {
+    padding-right: calc( 13px + #{var(--pfe-cta__arrow--Padding)} + 2px);
+    margin-right:  calc( -13px - #{var(--pfe-cta__arrow--Padding)} + 2px);
```
Comment (starryeyez024) [inline elements/pfe-cta/src/pfe-cta.scss]: also WIP

File: elements/pfe-cta/src/pfe-cta.scss [hunk 3]
```diff
@@ -9,65 +9,64 @@ $pfe-cta--Color--fallback: #003366;
   // regular variables
   --pfe-cta--BorderRadius: 0;
   --pfe-cta__arrow--Display:  inline;
-  --pfe-cta__arrow--Padding:  0 .125rem 0 .375rem;
+  --pfe-cta__arrow--Padding:  8px;
```
Comment (castastrophe) [inline elements/pfe-cta/src/pfe-cta.scss]: @starryeyez024 This doesn't seem related to the interpolation fix.

File: elements/pfe-cta/src/pfe-cta.scss [hunk 4]
```diff
@@ -9,65 +9,64 @@ $pfe-cta--Color--fallback: #003366;
...
-  border:           pfe-var(ui--border-width) pfe-var(ui--border-style) pfe-local(BorderColor);
+  background-color: #{pfe-local(BackgroundColor)};
+  border-radius:    #{pfe-local(BorderRadius)};
+  border:           #{pfe-var(ui--border-width)} #{pfe-var(ui--border-style)} #{pfe-local(BorderColor)};
 
   .pfe-cta--arrow {
-    display: pfe-local($cssvar: arrow--Display, $region: arrow);
-    padding: pfe-local($cssvar: Padding, $region: arrow);
+    display: #{pfe-local($cssvar: arrow--Display, $region: arrow)};
+    padding: 0;
```
Comment (castastrophe) [inline elements/pfe-cta/src/pfe-cta.scss]: Replacing the local variable with 0 seems out of scope.

File: elements/pfe-cta/src/pfe-cta.scss [hunk 5]
```diff
@@ -9,65 +9,64 @@ $pfe-cta--Color--fallback: #003366;
...
     fill: #{pfe-local(Color)};
     position: relative;
     top: .1rem; 
     width: 13px;
     height: 13px;
-    transition: .3s ease padding;
+    transition: .3s ease padding, .3s ease margin; // padding #{pfe-var(animation-timing)};
```
Comment (castastrophe) [inline elements/pfe-cta/src/pfe-cta.scss]: Update here seems out of scope

File: elements/pfe-cta/src/pfe-cta.scss [hunk 6]
```diff
@@ -108,13 +107,13 @@ $pfe-cta--Color--fallback: #003366;
...
   display: inline;
-  color: pfe-local(Color, $pfe-cta--Color--fallback) !important;  // because of redhat-theme styles on links
+  color: #{pfe-local(Color, $pfe-cta--Color--fallback)} !important;  // because of redhat-theme styles on links
   text-decoration: none;
   &::before {  // inner border on priority CTAs
     content: "";
     display: block;
     box-sizing: border-box;
-    border: 1px solid var(--pfe-cta__inner--BorderColor);
+    border: 1px solid transparent;
```
Comment (castastrophe) [inline elements/pfe-cta/src/pfe-cta.scss]: Removing the variable here also seems out of scope

File: elements/pfe-cta/src/pfe-cta.scss [hunk 7]
```diff
@@ -108,13 +107,13 @@ $pfe-cta--Color--fallback: #003366;
...
   display: inline;
-  color: pfe-local(Color, $pfe-cta--Color--fallback) !important;  // because of redhat-theme styles on links
+  color: #{pfe-local(Color, $pfe-cta--Color--fallback)} !important;  // because of redhat-theme styles on links
   text-decoration: none;
   &::before {  // inner border on priority CTAs
     content: "";
     display: block;
     box-sizing: border-box;
-    border: 1px solid var(--pfe-cta__inner--BorderColor);
+    border: 1px solid var(--pfe-cta__inner--BorderColor);;
```
Comment (castastrophe) [inline elements/pfe-cta/src/pfe-cta.scss]: There's a double colon at the end of this line. To use the mixin for this too: `#{pfe-local($cssvar: BorderColor, $region: inner)}`

Comment (starryeyez024) [reply elements/pfe-cta/src/pfe-cta.scss]: updated; thanks!

File: elements/pfe-cta/src/pfe-cta--lightdom.scss [hunk 8]
```diff
@@ -95,6 +91,9 @@ $LOCAL: cta;
 }
 
 @media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none) { //ie11
+  pfe-cta[pfe-priority="secondary"]  { //ie11
```
Comment (starryeyez024) [inline elements/pfe-cta/src/pfe-cta--lightdom.scss]: @castastrophe heads up I moved this style down here, and also modified lines 65 & 66. There was dark text on dark backgrounds in IE11, this resolves that issue

File: elements/pfe-cta/demo/index.html [hunk 9]
```diff
@@ -113,20 +114,24 @@ <h1>Old school testing (custom backgrounds)</h1>
...
-    <h2 style="color:white;">Custom dark band with broadcast vars</h2>
-    <pfe-cta><a href="#">Default (color responsive)</a></pfe-cta>
-    <pfe-cta pfe-priority="primary"><a href="#">Primary (color responsive)</a></pfe-cta>
-    <pfe-cta pfe-priority="secondary"><a href="#">Secondary (color responsive)</a></pfe-cta>
+    <h2 style="color:white;">Custom dark band with custom broadcast variables</h2>
+    <h3 style="color: white">Color responsive</h3>
+    <pfe-cta><a href="#">Default</a></pfe-cta>
+    <pfe-cta pfe-priority="primary"><a href="#">Primary</a></pfe-cta>
+    <pfe-cta pfe-priority="secondary"><a href="#">Secondary</a></pfe-cta>
+    <h3 style="color: white">Color override</h3>
```
Comment (starryeyez024) [inline elements/pfe-cta/demo/index.html]: just moved the descriptions about color override vs. color responsive out of the buttons and into headings

File: elements/pfe-cta/src/pfe-cta--lightdom.scss (hunk 8 above)
Comment (starryeyez024) [reply elements/pfe-cta/src/pfe-cta--lightdom.scss]: @LyndseyR can you move these updates to the lightdom CSS into your pfe-cta branch? then we can keep the scope here to just fixing functions

Comment (starryeyez024) [reply elements/pfe-cta/src/pfe-cta--lightdom.scss]: I pulled these changes out into https://github.com/patternfly/patternfly-elements/pull/569
